src/SFA.DAS.EmployerFinance.Api/App_Start/WebApiConfig.cs

```diff
@@ -50,6 +50,7 @@ public static void Register(HttpConfiguration config)
                 c.AddRegistry<RepositoriesRegistry>();
                 c.AddRegistry<TokenServiceRegistry>();
                 c.AddRegistry<DefaultRegistry>();
+                c.AddRegistry<StartupRegistry>();
```

Comment:
It's worth adding `DefaultRegistry` last to allow it to override any components if it needs to.